src/material-examples/button-types/button-types-example.css

```diff
@@ -3,3 +3,7 @@
   align-items: center;
   justify-content: space-around;
 }
+
+button, a {
```

Comment:
Do we want to make sure this `margin-right` is only added to ".example-button-row"?

Reply:
Sure. I went with the simpler selectors, because we've got style encapsulation, but these could still leak into other elements if we expand the example. It's changed now.